R/plot_with_settings.R

```diff
@@ -227,7 +229,13 @@ plot_with_settings_ui <- function(id) {
 #'     )
 #'   ),
 #'   server = function(input, output, session) {
-#'     plot_r <- reactive(ggplot2::qplot(data = faithful, x = waiting, y = eruptions))
+#'     plot_r <- plot_r <- reactive(
+#'       ggplot2::ggplot(
+#'         data = faithful,
+#'         ggplot2::aes(x = waiting, y = eruptions)
+#'       ) +
```

Comment:
Unless styler minds for some reason.

```suggestion
#'       ggplot2::ggplot(faithful, ggplot2::aes(x = waiting, y = eruptions)) +
```